examples/olsconfig.yaml

```diff
@@ -21,9 +21,13 @@ OLSConfig:
     library_log_level: warning
   default_provider: bam
   default_model: ibm/granite-13b-chat-v2
+Authentication:
+    ols_auth_check: False
```

Comment:
i added a developer config setting for this in #278, you just need to wire it up to your logic.